### PR TITLE
Displaying a help message in API store key generation - CURL requests

### PR DESCRIPTION
**Pull request by chamindias**
Fixing JIRA issue : https://wso2.org/jira/browse/APIMANAGER-3603

### CODE REVIEW

**Comment by chamindias** [comment]
Resolve conflicts in the fix.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
         "charsLeft": "Characters left: ",
 		"replyAdded": "Reply Successfully Added",
 		"callBackHelpMsg": "This parameter can be used in the authorization code grant type when invoking an API. For example, assume that both the client and the API Gateway are run on the same server and the Authorization API URL is https://localhost:8243/authorize. <strong> query component:</strong> “response_type=code&client_id=<consumer_key>&scope=PRODUCTION&redirect_uri=<application_callback_url>” and <strong>headers:</strong> Content-Type:application/x-www-form-urlencoded",
-		"tokenHelpMsg" : "The period after which the token will expire after generation. A negative value ensures that the token never expires."
+		"tokenHelpMsg" : "The period after which the token will expire after generation. A negative value ensures that the token never expires.",
+        "curlHelpMsg" : "CURL command created from the access tokens to send the request."
```
Comment (harsha89) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json]: slight formatting issue

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js [hunk 2]
```diff
@@ -236,6 +236,21 @@ $(document).ready(function () {
...
+    $('.curlHelp_popup_prod').click(function(){
+        $('#curlProdtoken_help').toggle('fast', function()
+        {
+            $('#curlProdtoken_help').html(i18n.t('info.curlHelpMsg'));
+        });
+        return false;
+    });
+
+    $('.curlHelp_popup_sand').click(function(){
+        $('#curlSandtoken_help').toggle('fast', function(){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js]: Formatting issues in above two lines

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js [hunk 3]
```diff
@@ -236,6 +236,21 @@ $(document).ready(function () {
 	    return false;
 	 });
 
+    $('.curlHelp_popup_prod').click(function(){
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js]: Formatting issue

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list-element/template.jag [hunk 4]
```diff
@@ -136,19 +136,23 @@
...
-											<div class="span2">
+											<div class="span12">
 												<div class="dropdown">
 													<button class="btn btn-default dropdown-toggle curlurl-genbtn" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-expanded="true"
 													    title="View cURL command to generate a token as an end user">
 														cURL <span class="caret"></span>
 													</button>
+
+													<a class="icon-question-sign curlHelp_popup_prod"></a>
+                                                    <div id="curlProdtoken_help"></div>
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list-element/template.jag]: Please fix the formatting issue

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json (hunk 1 above)
Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json]: Formatting is now corrected : https://github.com/wso2/carbon-apimgt/blob/release-1.10.x/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json#L20

Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json]: Formatting is corrected by now. Please see : https://github.com/wso2/carbon-apimgt/blob/release-1.10.x/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/locales/js/i18nResources.json#L22

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list-element/template.jag (hunk 4 above)
Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list-element/template.jag]: Done in PR https://github.com/wso2/carbon-apimgt/pull/1335

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js (hunk 3 above)
Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js]: Done in PR https://github.com/wso2/carbon-apimgt/pull/1335

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js (hunk 2 above)
Comment (chamindias) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js]: Done in PR https://github.com/wso2/carbon-apimgt/pull/1335

**Pull request merged**